control-plane-operator/hostedclusterconfigoperator/controllers/resources/rbac/reconcile.go

```diff
@@ -549,3 +549,38 @@ func ReconcileDeployerClusterRoleBinding(r *rbacv1.ClusterRoleBinding) error {
 	}
 	return nil
 }
+
+func ReconcileUserOAuthClusterRoleBinding(r *rbacv1.ClusterRoleBinding) error {
+	if r.Annotations == nil {
+		r.Annotations = map[string]string{}
+	}
+	r.Annotations["rbac.authorization.kubernetes.io/autoupdate"] = "true"
+	r.RoleRef = rbacv1.RoleRef{
+		APIGroup: rbacv1.SchemeGroupVersion.Group,
+		Kind:     "ClusterRole",
+		Name:     "system:openshift:useroauthaccesstoken-manager",
+	}
+	r.Subjects = []rbacv1.Subject{
+		{
+			Kind:     "Group",
+			APIGroup: rbacv1.SchemeGroupVersion.Group,
+			Name:     "system:authenticated:oauth",
+		},
+	}
+	return nil
+}
+
+func ReconcileUserOAuthClusterRole(r *rbacv1.ClusterRole) error {
+	if r.Annotations == nil {
+		r.Annotations = map[string]string{}
+	}
+	r.Annotations["rbac.authorization.kubernetes.io/autoupdate"] = "true"
```

Comment:
Why do we need this annotation?

Reply:
not sure if we do or why but Its present in the yaml file I used as a reference ([link](https://github.com/openshift/cluster-authentication-operator/blob/bebf0fd3932be12594227b415fecd5d664611bc0/bindata/oauth-apiserver/RBAC/useroauthaccesstokens_clusterrole.yaml)) aswell as on regular ocp .

Reply:
can we please investigate and clarify with that team why is needed?

Reply:
My vote is to leave it on there, given that at some point we may want to honor this annotation in the hcco so that we're in line with kube.